language/fr/lostcode.lang

```diff
@@ -0,0 +1,37 @@
+<?php
+
+include("{$_EnginePath}language/en/lostcode.lang");
```

Comment:
This file's translation is completed, therefore you can remove this default import.

Reply:
Done.